### ISSUE TITLE
Add rendering of note@stem.mod="#slash"

### ISSUE THREAD

**Issue by craigsapp** [rism-digital/verovio#1884]
Related to discussion in issues #1881 and  #716, `note@stem.mod="#slash"` should be rendered on notes in a similar manner to the current implementation of `sprech`.  There are 6 slashes:
     https://music-encoding.org/guidelines/v4/data-types/data.stemmodifier.html

Currently the slash stem modifications are ignored:

<img width="659" alt="Screen Shot 2020-12-15 at 8 09 00 AM" src="https://user-images.githubusercontent.com/3487289/102240523-d132da80-3eac-11eb-8753-c5e45e094057.png">

Expected rendering:

<img width="649" alt="Screen Shot 2020-12-15 at 8 11 13 AM" src="https://user-images.githubusercontent.com/3487289/102240820-22db6500-3ead-11eb-8c1a-beef268b6347.png">

(obviously 4-6 slashes would require lengthening the stems.)

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
 <meiHead>
  <fileDesc>
   <titleStmt>
    <title />
   </titleStmt>
   <pubStmt />
  </fileDesc>
  <encodingDesc>
   <appInfo>
    <application isodate="2020-12-15T08:06:28" version="3.1.0-dev-70bd0dc-dirty">
     <name>Verovio</name>
     <p>Transcoded from Humdrum</p>
    </application>
   </appInfo>
  </encodingDesc>
  <workList>
   <work>
    <title />
   </work>
  </workList>
 </meiHead>
 <music>
  <body>
   <mdiv xml:id="mdiv-0000001701894680">
    <score xml:id="score-0000001381532705">
     <scoreDef xml:id="scoredef-0000000875622581">
      <staffGrp xml:id="staffgrp-0000000018642638">
       <staffDef xml:id="staffdef-0000001048027346" n="1" lines="5">
        <clef xml:id="clef-0000001688312356" shape="G" line="2" />
       </staffDef>
      </staffGrp>
     </scoreDef>
     <section xml:id="section-L1F1">
      <measure xml:id="measure-L1" right="invis">
       <staff xml:id="staff-0000001696773026" n="1">
        <layer xml:id="layer-L1F1N1" n="1">
         <note xml:id="note-L2F1" stem.mod="sprech" dur="4" oct="4" pname="e" accid.ges="n" />
         <note xml:id="note-L4F1" stem.mod="1slash" dur="4" oct="4" pname="e" accid.ges="n" />
         <note xml:id="note-L6F1" stem.mod="2slash" dur="4" oct="4" pname="e" accid.ges="n" />
         <note xml:id="note-L10F1" stem.mod="3slash" dur="4" oct="4" pname="e" accid.ges="n" />
         <note xml:id="note-L18F1" stem.mod="4slash" dur="4" oct="4" pname="e" accid.ges="n" />
         <note xml:id="note-L34F1" stem.mod="5slash" dur="4" oct="4" pname="e" accid.ges="n" />
         <note xml:id="note-L66F1" stem.mod="6slash" dur="4" oct="4" pname="e" accid.ges="n" />
        </layer>
       </staff>
      </measure>
     </section>
    </score>
   </mdiv>
  </body>
 </music>
</mei>
```

If the slashes mean a tremolo, then the note should be enclosed in a `<bTrem>`, and the current semantic system that infers slashes based on the `bTrem@unitdur` should remain:

<img width="350" alt="Screen Shot 2020-12-15 at 8 05 21 AM" src="https://user-images.githubusercontent.com/3487289/102240039-4a7dfd80-3eac-11eb-91b0-8f38d3dee2fb.png">

In such cases `note@stem.mod` is optional and if given,  `note@stem.mod` should override the inferred slash count generated by `bTrem@unitdur`.

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
 <meiHead>
  <fileDesc>
   <titleStmt>
    <title />
   </titleStmt>
   <pubStmt />
  </fileDesc>
  <encodingDesc>
   <appInfo>
    <application isodate="2020-12-15T08:03:31" version="3.1.0-dev-70bd0dc-dirty">
     <name>Verovio</name>
     <p>Transcoded from Humdrum</p>
    </application>
   </appInfo>
  </encodingDesc>
  <workList>
   <work>
    <title />
   </work>
  </workList>
 </meiHead>
 <music>
  <body>
   <mdiv xml:id="mdiv-0000001732600802">
    <score xml:id="score-0000000916200983">
     <scoreDef xml:id="scoredef-0000000925394403">
      <staffGrp xml:id="staffgrp-0000002031037947">
       <staffDef xml:id="staffdef-0000001097076262" n="1" lines="5">
        <clef xml:id="clef-0000000884488278" shape="G" line="2" />
       </staffDef>
      </staffGrp>
     </scoreDef>
     <section xml:id="section-L1F1">
      <measure xml:id="measure-L1" right="invis">
       <staff xml:id="staff-0000000178868360" n="1">
        <layer xml:id="layer-L1F1N1" n="1">
         <bTrem xml:id="btrem-L3F1-L4F1" unitdur="8">
          <note xml:id="note-L3F1" dur="4" oct="4" pname="e" accid.ges="n" />
         </bTrem>
         <bTrem xml:id="btrem-L5F1-L8F1" unitdur="16">
          <note xml:id="note-L5F1" dur="4" oct="4" pname="e" accid.ges="n" />
         </bTrem>
         <bTrem xml:id="btrem-L9F1-L16F1" unitdur="32">
          <note xml:id="note-L9F1" dur="4" oct="4" pname="e" accid.ges="n" />
         </bTrem>
         <bTrem xml:id="btrem-L17F1-L32F1" unitdur="64">
          <note xml:id="note-L17F1" dur="4" oct="4" pname="e" accid.ges="n" />
         </bTrem>
         <bTrem xml:id="btrem-L33F1-L64F1" unitdur="128">
          <note xml:id="note-L33F1" dur="4" oct="4" pname="e" accid.ges="n" />
         </bTrem>
         <bTrem xml:id="btrem-L65F1-L128F1" unitdur="256">
          <note xml:id="note-L65F1" dur="4" oct="4" pname="e" accid.ges="n" />
         </bTrem>
        </layer>
       </staff>
      </measure>
     </section>
    </score>
   </mdiv>
  </body>
 </music>
</mei>
```

------------------------

For unmeasured tremolos, the encoding should be

```xml
 <bTrem form="unmeas">
     <note stem.mod="3slash">
</bTrem>
```

If there is no `note@stem.mod` for an unmeasured tremolo, then `note@stem.mod="3slash"` should be implied, since that is the most common symbol for an unmeasured tremolo.  So the following should be equivalent visually to the above example:

```xml
 <bTrem form="unmeas">
     <note>
</bTrem>
```

Currently tremolos are not realized in the MEI-to-MIDI conversion in MEI.  When they are implemented, `<bTrem>` would be required to realize a tremolo (not `note@stem.mod`).   For unmeasured tremolos, it would be useful to have a scoreDef or staffDef attribute that sets the duration of unmeasured tremolos in terms of seconds or milliseconds (this can be discussed more in the future).  In other words, unmeasured tremolos are independent of tempo and are played "as fast as possible".  An unmeasured tremolo could be calculated to be played with a duration of 80 milliseconds, for example, or the nearest number of equal divisions of the parent-note's duration at the given tempo.  So if the tempo is slow there would be many more repeated notes in the unmeasured tremolo as compared to playing the same note in a faster tempo.

There would be a question about what to do in this sort of case:

```xml
<bTrem>
    <note stem.mod="2slash">
</bTrem>
```

For notation, there should be a note with two slashes on it.  I would propose that this would be handled gracefully by a MIDI rendering by inferring `bTrem@unitdur` from the `note@stem.mod` contents.  So in this case `bTrem@unitdur="16"` would be inferred from `note@stem.mod="2slash"`.

In other words, `bTrem@unitdur` and `note@stem.mod="#slash"` would be equal in their importance.  Only one should be given, and the other could be inferred from it.  For notation, the slash count could be inferred from `bTrem@unitdur` but can be overridden if `note@stem.mod` is given.  Likewise for MIDI rendering, `note@stem.mod` could be used to infer `bTrem@unitdir`, but this can be overrideen if `bTrem@unitdur` is given.   This is somewhat similar to `note@accid` and `note@accid.ges`, but the parameter pair are located in separate elements.
